src/core/georeferencing.h

```diff
@@ -696,25 +712,34 @@ constexpr unsigned int Georeferencing::scaleFactorPrecision()
 	return 6u;
 }
 
+inline
+constexpr unsigned int Georeferencing::scaleFactorMultiplier()
+{
+	return 1000000.0;
+}
+
 inline
 double Georeferencing::roundScaleFactor(double value)
 {
-	// This must match the implementation in scaleFactorPrecision().
-	return floor(value*1000000.0+0.5)/1000000.0;
+	return floor(value*scaleFactorMultiplier()+0.5)/scaleFactorMultiplier();
 }
 
 inline
 constexpr unsigned int Georeferencing::declinationPrecision()
 {
-	// This must match the implementation in declinationRound().
 	return 2u;
 }
 
+inline
+constexpr unsigned int Georeferencing::declinationMultiplier()
+{
+	return 100.0;
```

Comment:
Possibly `1E2` in place of the `100.0` to align the style with the next patch.

Reply:
I expect to make this change.